kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -4,13 +4,13 @@
 from collections import defaultdict
 from time import time
 
+from six import string_types
+
+from datadog_checks.base import AgentCheck, ConfigurationError, is_affirmative
 from kafka import KafkaAdminClient, KafkaClient
```

Comment:
That's not great, because we have a folder named "kafka" isort does not realize that it is a 3rd party lib and that the import should be next to `from six import string_types`.

Would there be any nice way to trick isort ?

Reply:
Maybe using isort settings ? https://github.com/timothycrosley/isort/wiki/isort-Settings#full-reference-of-isort-settings